fdbserver/RestoreLoader.actor.cpp

```diff
@@ -463,25 +464,25 @@ ACTOR Future<Void> sendMutationsToApplier(VersionedMutationsMap* pkvOps, int bat
 				              nodeIDs.contents());
 				ASSERT(mvector.size() == nodeIDs.size());
 
-				if (debugMutation("RestoreLoader", commitVersion.version, kvm)) {
-					TraceEvent e("DebugSplit");
-					int i = 0;
-					for (auto& [key, uid] : *pRangeToApplier) {
-						e.detail(format("Range%d", i).c_str(), printable(key))
-						    .detail(format("UID%d", i).c_str(), uid.toString());
-						i++;
+				{
+					TraceEvent&& e = debugMutation("RestoreLoaderDebugSplit", commitVersion.version, kvm);
```

Comment:
Could you change the event name to `RestoreLoaderDebugRangeToAppliers`, which better reflects the event details.